public/javascripts/ood_shell.js

```diff
@@ -148,10 +100,17 @@ OodShell.prototype.onTerminalResize = function (columns, rows) {
   // React to size changes here.
   // Secure Shell pokes at NaCl, which eventually results in
   // some ioctls on the host.
-  this.socket.send(JSON.stringify({
-    resize: {
-      cols: columns,
-      rows: rows
-    }
-  }));
+  
+  // iff the socket is ready. race condition in FF where we try to resize before the socket is open
```

Comment:
This here can cause issues in Firefox, so I left a note.  There's definitely a chance for too much recursion, but that's only if users are crazily resizing. In either case, the browser eats it and eventually syncs.